system/velodyne_monitor/Readme.md

```diff
@@ -0,0 +1,63 @@
+# velodyne_monitor
+
+## Purpose
+
+This node monitors the status of Velodyne.
+The result of the status is published as diagnostics.
+
+## Inner-workings / Algorithms
+
+The status of Velodyne is get from `https://[ip_address]/cgi/{info, settings, status, diag}.json`.
```

Comment:
```suggestion
The status of Velodyne LiDARs can be retrieved from `https://[ip_address]/cgi/{info, settings, status, diag}.json`.
```
IMO: Since Velodyne is a company name, add LiDARs or sensors might be better?

Reply:
129f0b5